### PR TITLE
Enable logout

### LINKED ISSUE #46: Enable users to logout
Following:

https://flask-login.readthedocs.io/en/latest/#login-example

Looks like a piece of cake to just add a new route...

```python
@app.route("/logout")
@login_required
def logout():
    logout_user()
    return redirect(somewhere)
```

...but we may need to also do oauth things with flask-dance to tell Google that the user is logged out of our app. 

Based on the flask-dance docs, I think we don't need to do anything with Google.
https://flask-dance.readthedocs.io/en/latest/logout.html#log-out-local-account

The whole section on logging out is a good read.

### PR DESCRIPTION
**Pull request by rmarow**
Resolves #46

### CODE REVIEW

File: usaon_vta_survey/templates/macros/profile_or_login_button.j2
Comment (rmarow) [inline usaon_vta_survey/templates/macros/profile_or_login_button.j2:1]: Looking for inspiration on renaming this macro now that it also includes `logout`.

Comment (MattF-NSIDC) [reply usaon_vta_survey/templates/macros/profile_or_login_button.j2:1]: Hmm... Maybe just a super wordy name? I'm sure we could do better, but I think extra verbosity is a good default :)

`profile_logout_button_or_login_button`?

**Review by MattF-NSIDC — APPROVED** [review]

File: usaon_vta_survey/templates/macros/profile_or_login_button.j2
Comment (MattF-NSIDC) [reply usaon_vta_survey/templates/macros/profile_or_login_button.j2:1]: `user_login_management_buttons`?

Comment (rmarow) [reply usaon_vta_survey/templates/macros/profile_or_login_button.j2:1]: Gonna go with `user_login_management_buttons` :) changing that now

**Pull request merged**